Core/ExpectedException/ExpectedExceptionMethodBodyTransformer.cs

```diff
@@ -29,26 +29,12 @@ public class ExpectedExceptionMethodBodyTransformer : ISyntaxTransformer<MethodD
   {
     public MethodDeclarationSyntax Transform (MethodDeclarationSyntax node, IExpectedExceptionModel model)
     {
-      if (node.Body == null || node.Body.Statements.Count == 0)

```

Comment:
I believe we have reached a good compromise between readability and usage safety, for our use cases it should more than do the trick